colorbleed/plugins/fusion/inventory/select_loader_color.py

```diff
@@ -0,0 +1,40 @@
+from avalon import api
+from avalon.vendor.Qt import QtGui, QtWidgets
+
+import avalon.fusion
+
+
+class FusionSelectLoaderColor(api.InventoryAction):
+    """Update the color of the selected tools"""
+
+    label = "Select Loader Color"
+    icon = "plus"
+    color = "#d8d8d8"
+
+    def process(self, containers):
+        """Color all selected tools the selected colors"""
+
+        comp = avalon.fusion.get_current_comp()
+
+        # Launch pick color
+        color = QtGui.QColor(1.0, 1.0, 1.0)
```

Comment:
Can we have the "start color" in the UI start with the currently set color, for example from the first container or the last. Just seems odd that it starts with a pure white even if you want to update the color of a single container.

Or what's possible in that regard?

Reply:
I did have that before but I thought it was a nice thing to have but not really the main goal of the action. Now that I pass on a `QColor` instance it makes things easier.